logic/boolean/spell.yaml

```
@@ -2,12 +2,17 @@ __use_yte__: true
 
 custom: |
   ?f"""function(value) {{
-    const trueLabel = '<span style="color: #1f77b4; font-weight: bold;">+</span>';
-    const falseLabel = '<span style="color: #d62728; font-weight: bold;">-</span>';
+    const trueLabel = '<span style=" display: inline-block; padding: 0.5em 1em; font-size: 0.875em; font-weight: bold; color: white; background-color: rgb(0,128,255); border-radius: 0.25em; text-align: center;">+</span>';
+
+    const falseLabel = '<span style=" display: inline-block; padding: 0.5em 1em; font-size: 0.875em; font-weight: bold; color: white; background-color: rgb(255,0,0); border-radius: 0.25em; text-align: center;">-</span>';
+    
+    const na_value = "";
     if (value === "{true_value}") {{
         return trueLabel;
     }} else if (value === "{false_value}") {{
         return falseLabel;
+    }} else if (value === "{na_value}"){{
```

```suggestion
    }} else if (value === na_value){{
```

`"{na_value}"` would actually mean a variable that is rendered by yte. As you want to refer to the javascript constant the quotes and curly braces need to go.

---

@fxwiegand I am still thinking about how na values are best handled.
Can this be done implicitly without defining a variable `na_value`.
While it makes sense to defined the `true_value` and `false_value` I would expect the `na_value` to always be an empty entry. So in my opinion this should be hard coded.
I also tried this locally but the empty lines in the tsv-files are always ignored and seem to be skipped. Is this somehow datavzrd related?

---

Yes, that was my fault. I totally missed this and realized too late that NAs are handled implicitly.
@BenOrl Sorry for that. I guess if you remove the na_value lines once again we are good to go.